src/index.html

```diff
@@ -1,14 +1,67 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible"
+          content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet"
+          href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home"
+         class="logo">
+        <img src="../src/images/logo.png"
+             alt="MOYO-logo">
+      </a>
```

Comment:
```suggestion
      <a
        href="#home"
        class="logo"
      >
        <img
          src="../src/images/logo.png"
          alt="MOYO-logo"
        >
      </a>
```

Reply:
<img width="1095" alt="Screenshot 2023-07-17 at 15 24 58" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/0a2f82ec-ee74-40fa-9b10-559468c05fd6">
Fix similar problems in all code